test/runtests.jl

```diff
@@ -284,6 +289,12 @@ end
         @test FASTQ.hasquality(record)
         @test FASTQ.quality(record) == b"@BCFFFDFHHHHHJJJIJIJJIJJJJJJJJIJJJJIIIJJJIJJJ" .- 33
 
+        record = FASTQ.Record("id", "desc", "AAGCT", collect("@BCFF"))
+        record2 = FASTQ.Record("id", "desc", "AAGCT", collect("@BCFF"))
```

Comment:
Same comment as above, add some tests for inequality, and include records that lack a description.